python-checks/src/main/java/org/sonar/python/checks/VerifiedSslTlsCertificateCheck.java

```diff
@@ -318,6 +320,12 @@ private static void standardSslCheck(SubscriptionContext subscriptionContext) {
       }));
   }
 
+  private static void standardSslCheckForRegularArgument(SubscriptionContext subscriptionContext) {
+    var argument = (RegularArgument) subscriptionContext.syntaxNode();
+    searchRhsForVulnerableMethod(argument.expression())
```

Comment:
_Nitpick:_ I think we can rename this method to `isVulnerableMethodCall` since it is no longer only looking for RHS of assignments.